include/GafferSceneUI/LightVisualiser.h

```diff
@@ -65,7 +67,7 @@ class GAFFERSCENEUI_API LightVisualiser : public IECore::RefCounted
 
 		/// Must be implemented by derived classes to visualise
 		/// the light contained within `shaderNetwork`.
-		virtual IECoreGL::ConstRenderablePtr visualise( const IECore::InternedString &attributeName, const IECoreScene::ShaderNetwork *shaderNetwork, IECoreGL::ConstStatePtr &state ) const = 0;
+		virtual IECoreGL::ConstRenderablePtr visualise( const IECore::InternedString &attributeName, const IECoreScene::ShaderNetwork *shaderNetwork, IECoreGL::ConstStatePtr &state, const IECore::CompoundObject *attributes ) const = 0;
```

Comment:
I think readability would be aided by keeping the `state` output parameter as the last one - "using these inputs, set this output".

Reply:
Sorry, I completely overlooked the input/output separation. It's fixed in cfd8c56